setup.cfg

```diff
@@ -30,6 +30,7 @@ exclude = .git,
     great_expectations/expectations,
     great_expectations/dataset,
     great_expectations/render,
+    scripts/*,
```

Comment:
started getting linting errors due to the complexity on this file.
Have we been ignoring this error?